site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -56,6 +56,40 @@ The FusionAuth Id of the User that is being linked to the identity provider.
 include::docs/src/json/identity-providers/links/create-link-request.json[]
 ----
 
+=== Request
+
```

Comment:
So this is the preferred method (to wrap everything in the JSON in an `identityProviderLink` object). We should put it first and add deprecated to the other call request method. I don't know if we have an example of an entire call being deprecated, so I'd wing it with what looks good. (you mentioned this in the description of this PR, but it needs to be done before you release it).

I think you should mark the old ones as deprecated, not remove them, since the new method would break with versions before 1.43.

I was confused by the existence of two of the same methods, but now I get it. You might even add a note in front of the old ones, saying something like "This method is functionally equivalent to the above method, but the above method is more consistent with other APIs. Use of this method is deprecated."

You also need to add a `since` field to the new API options.

Reply:
![image](https://user-images.githubusercontent.com/12218379/221269644-8496f5a5-db0d-4a3f-a6e3-50789f96bcf6.png)